src/dune_rules/install_rules.ml

```diff
@@ -560,11 +589,22 @@ end = struct
             and* modules =
               Dir_contents.ocaml dir_contents
               >>| Ml_sources.modules ~for_:(Library name)
+            and* melange_runtime_deps =
+              match Lib_info.melange_runtime_deps info with
+              | External _paths -> assert false
```

Comment:
I wonder if we could read from `Lib.t` instead to remove this assertion.

Reply:
I believe this is related to how we represent stuff in `Lib_info`. there's other code that shares the same behavior. I don't think we can put this in `Lib.t` because it comes from the dune file, which creates a Lib_info.t anyway.